pkg/cr/cr.go

```diff
@@ -76,7 +78,7 @@ func (c *CRUtil) updateCR(object *athenz_domain.AthenzDomain, newCR *athenz_doma
 	newObjCopy.Signature = ""
 	newObjCopy.Domain.Policies.Signature = ""
 	eql := reflect.DeepEqual(oldObjCopy, newObjCopy)
-	if eql {
+	if eql && object.Status.Message == "" {
```

Comment:
Can we also have a reflect deep equal check for the status object in case we add more fields there in the future?